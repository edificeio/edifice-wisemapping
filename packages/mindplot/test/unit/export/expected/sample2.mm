<map version="1.0.1">
    <node ID="ID_null" LINK="prospace.com" TEXT="Prospace">
        <edge COLOR="#cc5627"/>
        <richcontent TYPE="NOTE">
            <html xmlns="http://www.w3.org/1999/xhtml">
                <head><head/>
                <body>
                    <p/>
                </body>
            </html>
        </richcontent>
        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Third Party ">
            <node ID="ID_null" LINK="thumbshots.org" POSITION="left" STYLE="rectangle" TEXT="Thumbshot"/>
        </node>
        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Newspapers">
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="College Newspapers"/>
        </node>
        <node ID="ID_null" POSITION="right" STYLE="fork" TEXT="Partnerships">
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Websites">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Business Networks">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="konnects.com"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Xing"/>
                </node>
            </node>
        </node>
        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Career Counseling">
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Professional Associations"/>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Fraternity"/>
        </node>
        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="White-Label">
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="http://www.harrisconnect.com/"/>
        </node>
    </node>
</map>