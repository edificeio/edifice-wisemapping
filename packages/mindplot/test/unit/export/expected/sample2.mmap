<?xml version="1.0" encoding="UTF-8"?><ap:Map xmlns:ap="http://schemas.mindjet.com/MindManager/Application/2003"><ap:OneTopic><ap:Topic OId="nullnezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="nullnezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="nullnezC90m8NYAi2fjQvw=="><ap:Text PlainText="Thumbshot"><ap:Font/></ap:Text><ap:SubTopicShape/><ap:Hyperlink Url="thumbshots.org"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Third Party "><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="nullnezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="nullnezC90m8NYAi2fjQvw=="><ap:Text PlainText="College Newspapers"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Newspapers"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="nullnezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="nullnezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="nullnezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="nullnezC90m8NYAi2fjQvw=="><ap:Text PlainText="konnects.com"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="nullnezC90m8NYAi2fjQvw=="><ap:Text PlainText="Xing"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Business Networks"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Websites"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Partnerships"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="nullnezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="nullnezC90m8NYAi2fjQvw=="><ap:Text PlainText="Professional Associations"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="nullnezC90m8NYAi2fjQvw=="><ap:Text PlainText="Fraternity"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Career Counseling"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="nullnezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="nullnezC90m8NYAi2fjQvw=="><ap:Text PlainText="http://www.harrisconnect.com/"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="White-Label"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Prospace"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/><ap:SubTopicsShape SubTopicsAlignment="urn:mindjet:Center" SubTopicsGrowth="urn:mindjet:Horizontal" SubTopicsGrowthDirection="urn:mindjet:AutomaticHorizontal&#10;                        " VerticalDistanceBetweenSiblings="150"/><ap:Hyperlink Url="prospace.com"/></ap:Topic></ap:OneTopic><ap:Relationships/></ap:Map>