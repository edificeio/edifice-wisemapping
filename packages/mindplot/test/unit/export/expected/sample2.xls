<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<?mso-application progid="Excel.Sheet"?><Workbook xmlns:x="urn:schemas-microsoft-com:office:excel" xmlns:ss="urn:schemas-microsoft-com:office:spreadsheet" xmlns:duss="urn:schemas-microsoft-com:office:dummyspreadsheet" xmlns:o="urn:schemas-microsoft-com:office:office" xmlns="urn:schemas-microsoft-com:office:spreadsheet">
<Styles>
<Style ss:ID="s16" ss:Name="attribute_cell">
<Borders>
<Border ss:Position="Bottom" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Left" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Right" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Top" ss:LineStyle="Continuous" ss:Weight="1"/>
</Borders>
</Style>
<Style ss:ID="s17" ss:Name="attribute_header">
<Borders>
<Border ss:Position="Bottom" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Left" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Right" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Top" ss:LineStyle="Continuous" ss:Weight="1"/>
</Borders>
<Font ss:Bold="1"/>
</Style>
</Styles>
<Worksheet ss:Name="FreeMind Sheet">
<Table>
<Row>
<Cell ss:Index="1">
<Data ss:Type="String">Prospace</Data>
<Comment>
<ss:Data xmlns="http://www.w3.org/TR/REC-html40">
<p/>
</ss:Data>
</Comment>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Third Party </Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Thumbshot</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Newspapers</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">College Newspapers</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Partnerships</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Websites</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">Business Networks</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="5">
<Data ss:Type="String">konnects.com</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="5">
<Data ss:Type="String">Xing</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Career Counseling</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Professional Associations</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Fraternity</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">White-Label</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">http://www.harrisconnect.com/</Data>
</Cell>
</Row>
</Table>
</Worksheet>
</Workbook>
