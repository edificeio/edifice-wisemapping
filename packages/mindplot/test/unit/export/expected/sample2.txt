1 Prospace
	* Link: prospace.com
	* Note: 
	1.1 Third Party 
		1.1.1 Thumbshot
			* Link: thumbshots.org
	1.2 Newspapers
		1.2.1 College Newspapers
	1.3 Partnerships
		1.3.1 Websites
			1.3.1.1 Business Networks
				1.3.1.1.1 konnects.com
				1.3.1.1.2 Xing
	1.4 Career Counseling
		1.4.1 Professional Associations
		1.4.2 Fraternity
	1.5 White-Label
		1.5.1 http://www.harrisconnect.com/
